doc/qml-gui.md

```diff
@@ -0,0 +1,15 @@
+# QML-BASED GUI BUILD NOTES
```

Comment:
can you drop this change, i will include it in my doc pr

Reply:
Done.